app/assets/javascripts/discourse/tests/integration/components/widgets/user-menu-test.js

```diff
@@ -92,16 +92,19 @@ module("Integration | Component | Widget | user-menu", function (hooks) {
   });
 
   test("private messages - disabled", async function (assert) {
-    this.currentUser.setProperties({ admin: false, moderator: false });
-    this.siteSettings.personal_message_enabled_groups = "13"; // trust_level_3 auto group ID;
+    this.currentUser.setProperties({
+      admin: false,
+      moderator: false,
+      can_send_private_messages: false,
+    });
 
     await render(hbs`<MountWidget @widget="user-menu" />`);
 
     assert.ok(!exists(".user-pms-link"));
   });
 
   test("private messages - enabled", async function (assert) {
-    this.siteSettings.personal_message_enabled_groups = "11"; // trust_level_1 auto group ID;
+    this.currentUser.set("can_send_private_messages", true);
```

Comment:
Noticed there's the 

```
      admin: false,
      moderator: false,
      ...
```

above. Do we need it here to be explicit that `admin` is not required?

Reply:
Good point, will fix